dashboard-app/src/features/dashboard/Dashboard/useDashboardStatistics.ts

```diff
@@ -63,7 +64,7 @@ export default () => {
     // most active months (12 months)
     setTimeStats(
       findMostActive(
-        collectBy((log) => moment(log.startedAt).format(Months.format.verbose), fullLogs)
+        collectBy((log) => moment(log.startedAt).format(Months.format.abreviated), fullLogs)
```

Comment:
Why has this been changed?

Reply:
This is the date format specified in the user stories (010A/B/C)